### DISCUSSION TITLE
LangFuse Low Level SDK does not render images by passing the link

### DISCUSSION THREAD

**Discussion by VikramShenoy97** [langfuse/langfuse#3477] [Support]
Hello

I'm trying to figure out how to use the Low level Python SDK to render images without having a dependency on the OpenAI SDK. What should be the input/output format when I call trace.update() to render the images?

**Comment by marcklingen** [comment]
Happy to help. You can either implement the openai content parts schema explained [here](https://langfuse.com/docs/tracing-features/multi-modality) or include the image as a markdown image `![alt](src)` which will also be rendered in the UI.

Can you share a link to an example trace that does not work for you?
